test/Pattern/TestAsset/FailableCallback.php

```diff
@@ -2,7 +2,7 @@
 
 namespace LaminasTest\Cache\Pattern\TestAsset;
 
-class FailableCallback
+final class FailableCallback
```

Comment:
final is not needed in test assets IMO as they can only be used in this package

Reply:
Its not needed but wont hurt so we can keep it.
final aaaall the things.